content/en/docs/Creating derivatives/Packer/build_gcp.md

```diff
@@ -92,9 +92,9 @@ template. These are some of the relevant ones:
 * `gcp_cos_deploy_args`: This the command that will be executed once the
   Vanilla image booted. In this stage it is expected that user sets a command
   to install the desired cOS or derivative image. By default it is set to
-  `cos-deploy` which will deploy the latest cOS image in cOS repositories.
+  `elemental reset` which will deploy the cOS image from recovery partition.
```

Comment:
NIT: `from the` maybe? sounds better to me, but not sure if its the correct way.